src/Services/Match/Match.Presentation/Controllers/LikesController.cs

```diff
@@ -0,0 +1,21 @@
+using Match.Application.DTOs.Like.Request;
+using Match.Application.UseCases.LikeUseCases.Commands.Create;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Match.Presentation.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class LikesController(IMediator _mediator) : ControllerBase
+{
+    [HttpPost]
+    public async Task<IActionResult> CreateLike([FromBody]CreateLikeDto dto, CancellationToken cancellationToken)
```

Comment:
Creating a like doesn't sound good. Let's try to use something more appropriate here.